packages/pn-pa-webapp/src/api/external-registries/__test__/External-registries.api.test.ts

```diff
@@ -18,4 +19,23 @@ describe('External registries api tests', () => {
     mock.reset();
     mock.restore();
   });
+
+  it('getInstitutions', async () => {
+    const mock = new MockAdapter(apiClient);
```

Comment:
move the mock creation and reset/restore to the beforeAll, afterAll ecc..

check other tests